app/quest-boost/claim/[boostId]/page.tsx

```diff
@@ -0,0 +1,207 @@
+              {boost &&
+              boost?.winner &&
+              hexToDecimal(boost?.winner) === hexToDecimal(address) ? (
+                <>
+                  <div className={styles.token_logo}>
+                    {getTokenName() === "USDC" ? (
+                      <CDNImage
+                        src={"/icons/usdc.svg"}
+                        priority
+                        width={97}
+                        height={97}
+                        alt="usdc icon"
+                      />
+                    ) : (
+                      <CDNImage
+                        src={"/icons/eth.svg"}
+                        priority
+                        width={97}
+                        height={97}
+                        alt="usdc icon"
+                      />
+                    )}
+                  </div>
+                  <div className={styles.claim_button_text}>
+                    <p className={styles.claim_amount}>{boost?.amount}</p>
+                  </div>
+                  <div className={styles.token_symbol_container}>
+                    <div className="bg-[#1F1F25] flex-1 rounded-[12px] flex justify-center items-center">
+                      {getTokenName()}
+                    </div>
+                  </div>
+                </>
+              ) : (
+                <>
+                  <div className={styles.token_logo}>
+                    <img
+                      src="/visuals/animals/tiger.webp"
+                      height={256}
+                      width={254}
+                      alt="error image"
+                    />
+                  </div>
+                  <div className={styles.claim_button_text}>
+                    <p
+                      className="pb-[1.5rem] text-center"
+                      style={{ fontSize: 24 }}
+                    >
+                      Sorry, no win this time.
+                    </p>
+                  </div>
+                </>
+              )}
+            </div>
+
+            {boost &&
+            boost?.winner &&
+            hexToDecimal(boost?.winner) === hexToDecimal(address) ? (
+              <div className={styles.claim_button_animation}>
+                <Button
+                  disabled={
+                    boost?.claimed ||
+                    hexToDecimal(boost?.winner ?? "") !== hexToDecimal(address)
+                  }
+                  onClick={handleClaimClick}
+                >
+                  Collect my reward
+                </Button>
+              </div>
+            ) : null}
+          </>
+        ) : null}
+      </div>
+      {boost && displayLottie ? (
+        <div className="absolute ml-auto mr-auto left-0 right-0 flex justify-center w-full">
+          <Lottie
+            onEnterFrame={() => {
+              setDisplayCard(true);
+            }}
+            onComplete={() => {
+              setDisplayLottie(false);
+            }}
```

Comment:
`() => setDisplayLottie(false)` is not possible ? same on top

Reply:
done